release-pr/README.md

```diff
@@ -0,0 +1,140 @@
+# Neon Release PR
+
+A GitHub Action and CLI tool for creating and managing release PRs for Neon components.
+
+## Overview
+
+The `neon-release-pr` tool automates the process of creating release pull requests for Neon components. It handles the creation of release branches, cherry-picking commits, crafting merge commits, and setting up GitHub PR options like auto-approval and auto-merge.
+
+## GitHub Action Usage
+
+### Example 1: Interactive Workflow with Inputs
+
+Add the action to your workflow with user inputs:
+
+```yaml
+name: Create Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      component:
+        description: 'Component to release'
+        required: true
+        type: string
+      hotfix:
+        description: 'Create a hotfix release'
+        required: false
+        type: boolean
+        default: false
+      cherry_pick:
```

Comment:
nit: can we use dashes here as well, like `cherry-pick`?
as it's more common way to name vars in github actions